packages/inventory-compliance/src/SystemPolicyCard/SystemPolicyCard.js

```diff
@@ -1,5 +1,5 @@
-import React, { Fragment } from 'react';
-import { FormattedRelative, FormattedRelativeTime } from 'react-intl';
+import React from 'react';
+import DateFormat from '@redhat-cloud-services/frontend-components/DateFormat';
```

Comment:
Observation: I'm not fully sure whether this could use the `/esm` variant.

/cc @karelhala

Reply:
@Hyperkid123 Is working on eliminating the need for that as far as i know.

Reply:
If a package and app is using v3 of the components dependency, there is no need for the /esm partial.